src/domain/journey/challenge/pages/ChallengeDashboardPage.tsx

```diff
@@ -40,6 +41,8 @@ const ChallengeDashboardPage: FC<ChallengeDashboardPageProps> = ({ dialog }) =>
 
   const hasExtendedApplicationButton = useMediaQuery((theme: Theme) => theme.breakpoints.up('sm'));
 
+  const shareUrl = buildUpdatesUrl({ spaceNameId: spaceNameId ?? '', challengeNameId });
```

Comment:
normally, instead of `?? ''` with the spaceNameId, what we are doing is to check if we are in a Space and throw an error otherwise:
```
if (!spaceNameId) {
    throw new Error('Must be within a Space route.');
}
```